File: tests/omp/ordered3.cpp
void work(int i) {}
void ordered_good(int n)
{
  for (int i=0; i<n; i++) {
    if (i <= 10) {
         work(i);
    }
    if (i > 10) {
        work(i+1);
    }
  }
}

File: tests/omp/ordered3.ml
open Optitrust
open Target

let _ = Run.script_cpp (fun _ ->

  !! Omp.for_ [Ordered_c 0] [tBefore; cFor_c "i"];
  !! Omp.ordered  [] [tBefore; tIndex ~nb:2 0; cFun "work"];
  !! Omp.ordered  [] [tBefore; tIndex ~nb:2 1; cFun "work"];
)
